python/whylogs/experimental/api/logger/__init__.py

```diff
@@ -131,18 +131,39 @@ def _calculate_average_precisions(
     return averages
 
 
+def _all_strings(data: pd.Series) -> bool:
```

Comment:
wondering if pandas methods could be a little more efficient, like: https://stackoverflow.com/a/43191959/8830597

Reply:
There might be a better way to check, but the technique mentioned in statckflow applies to the full dataframe :(